tests/protractor/conf.js

```diff
@@ -1,61 +1,70 @@
-var _ = require('underscore'),
-    utils = require('./utils'),
-    spawn = require('child_process').spawn,
-    environment = require('./auth')(),
-    api;
+const _ = require('underscore'),
+      utils = require('./utils'),
+      spawn = require('child_process').spawn,
+      environment = require('./auth')(),
+      modules = [];
 
 _.templateSettings = {
   interpolate: /\{\{(.+?)\}\}/g
 };
 
-var login = function(browser) {
-  var loginUrlTemplate = _.template('http://{{apiHost}}:{{apiPort}}/{{dbName}}/login?redirect=');
-  var redirectUrl = encodeURIComponent('/' + environment.dbName  + '/_design/medic/_rewrite/#/messages?e2eTesting=true');
+const login = browser => {
+  const loginUrlTemplate = _.template('http://{{apiHost}}:{{apiPort}}/{{dbName}}/login?redirect=');
+  const redirectUrl = encodeURIComponent('/' + environment.dbName  + '/_design/medic/_rewrite/#/messages?e2eTesting=true');
   browser.driver.get(loginUrlTemplate(environment) + redirectUrl);
   browser.driver.findElement(by.name('user')).sendKeys(environment.user);
   browser.driver.findElement(by.name('password')).sendKeys(environment.pass);
   browser.driver.findElement(by.id('login')).click();
   // Login takes some time, so wait until it's done.
-  return browser.driver.wait(function() {
+  return browser.driver.wait(() => {
     return browser.driver.isElementPresent(by.css('body.bootstrapped'));
```

Comment:
Yeah, you're right. I tried to get rid of it but it looks really bad because that function is just the first parameter of three. I've pulled it out as a const function.